src/luascript.cpp

```diff
@@ -2433,6 +2433,9 @@ void LuaScriptInterface::registerFunctions()
 	registerMethod("Player", "getBankBalance", LuaScriptInterface::luaPlayerGetBankBalance);
 	registerMethod("Player", "setBankBalance", LuaScriptInterface::luaPlayerSetBankBalance);
 
+	registerMethod("Player", "hasStorage", LuaScriptInterface::luaPlayerHasStorage);
```

Comment:
These should be `hasStorageValue` and `clearStorageValue` instead, to match `get` and `set`